CMakeLists.txt

```diff
@@ -58,7 +58,7 @@ target_include_directories(
 # Add an alias to public name.
 add_library( bfg::Lyra ALIAS lyra )
 
-set(CMAKE_CONFIG_DEST "${CMAKE_INSTALL_LIBDIR}/cmake/lyra")
+set(CMAKE_CONFIG_DEST "${CMAKE_INSTALL_LIBDIR_ARCHIND}/cmake/lyra")
```

Comment:
Not a big fan of making things ArchIndependent right now. In future, the library might become non-header only for whatever reason.